pkg/analyzer/detection/detection_test.go

```diff
@@ -88,8 +88,26 @@ OSTREE_VERSION='49.84.202212201621-0'
 			want: []string{
 				"rhel-8-for-x86_64-baseos-eus-rpms__8_DOT_4",
 				"rhel-8-for-x86_64-appstream-eus-rpms__8_DOT_4",
+				"rhel-8-for-x86_64-nfv-tus-rpms__8_DOT_4",
 				"fast-datapath-for-rhel-8-x86_64-rpms",
 				"rhocp-4.9-for-rhel-8-x86_64-rpms",
+				"advanced-virt-for-rhel-8-x86_64-eus-rpms",
+			},
+		},
+		{
+			name: "when ocp/4.7 then should not use advanced-virt",
+			osReleaseContents: `
+ID="rhcos"
+VERSION_ID="4.7"
+OPENSHIFT_VERSION="4.7"
+RHEL_VERSION="8.4"
```

Comment:
can we add another test to ensure we can correctly get something like `8_DOT_3` or something other than 4 as well?

Reply:
```suggestion
RHEL_VERSION="8.3"
```

Reply:
@RTann like this?

Reply:
yeah as long as we can confirm that it doesn't just happen to work for 8.4 only